demo.py

```diff
@@ -68,11 +78,26 @@ def main():
         options = here.show_exits().keys()
         directions = ", ".join(options)
 
-        print(f"It's time to move on, {player.name}, your options are: {directions}")
+        print(f"It's time to move on, {player.name}!\nYour options are: {directions}")
         direction = input("WHERE WILL YOU GO?\n> ")
 
         ok = False
         while not ok:
+            if direction.lower() == "quit" or direction.lower() == "exit":
+                print("Your vision goes blank as you collapse to the ground.")
+                sys.exit()
+
+            area = 9999
+            try:
+                area = int(str(direction))
+            except:
+                print(direction)
+            if area <= state.count_locations():
+                    here = state.set_active_location(area)
+                    print(f"You fast travel to area {area}")
+                    ok = True
+                    continue
```

Comment:
This allows us to type in a number and travel to that location ID directly (instead of navigating through the connection points). It's a tad sloppy but it helped during debugging.